src/parser/expression.js

```diff
@@ -171,8 +175,18 @@ break;
         operands.push(constant(ctx));
         break;
       case Syntax.Identifier:
-        eatFunctionCall = true;
-        operands.push(maybeIdentifier(ctx));
+        const prev = last(operators);
+        if (prev && prev.value === ".") {
+          operators.pop();
+          operators.push({ ...prev, value: "[" });
+          operands.push(accessIdentifier(ctx));
+          eatUntil(isLSqrBracket);
+          consume();
+          eatFunctionCall = false;
```

Comment:
I'd like us to avoid doing this. Feel free to email me if you're stuck/need some more info on the parser implementation.